consensus/istanbul/backend/handler.go

```diff
@@ -49,18 +53,24 @@ func (sb *backend) Protocol() consensus.Protocol {
 
 func (sb *backend) decode(msg p2p.Msg) ([]byte, common.Hash, error) {
 	var data []byte
-	if err := msg.Decode(&data); err != nil {
-		return nil, common.Hash{}, errDecodeFailed
+	if sb.IsQBFTConsensus() {
```

Comment:
Each node will move to QBFT when the `qbftBlock` fork block is reached, it may happen that when a node starts processing QBFT messages, it may receive a IBFT message, which will result in an error decoding the message and the message will be ignored.